docs/proposals/observability-management.md

```diff
@@ -0,0 +1,82 @@
+* [Observability Management](#observability-management)
+  * [Motivation](#motivation)
+    * [Goals](#goals)
+  * [Proposal](#proposal)
+  * [Design Details](#design-details)
+    * [Monitoring Metrics](#monitoring-metrics)
+    * [Collecting Logs](#collecting-logs)
+    * [Display](#display)
+  * [Key Deliverable](#key-deliverable)
+  * [Road Map](#roadmap)
+
+# Observability Management
+
+## Motivation
+Currently, users can only check the status, parameters and metrics of tasks via the command line after creating edge-cloud synergy AI tasks by sedna.
+
+This proposal provides observability management for displaying logs and metrics to monitor tasks in real time, so that users can easily check the status, parameters and metrics of tasks.
```

Comment:
Lack of algorithm metrics to evaluate the performance of pradigms.

Reply:
The new version seems to have revised the comment, e.g.,  AccuracyMetricForNewModel